generic-ec-core/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "generic-ec-core"
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
Need to bump the required version in `generic-ec`, since you use the new function

Reply:
I always forget to do that...

Reply:
Updated

Reply:
I don't see the update in generic-ec/Cargo.toml

Reply:
I forgot to push it. Done